src/elements/content-explorer/ContentExplorer.js

```diff
@@ -763,8 +760,16 @@ class ContentExplorer extends Component<Props, State> {
         this.unselect();
         item.selected = true;
 
+        // ensure that only selected item is marked as selected in currentCollection
+        const newCollection: Collection = { ...this.state.currentCollection };
+        if (newCollection.items) {
+            for (let i = 0; i < newCollection.items.length; i += 1) {
```

Comment:
Looks like a good opportunity for `map`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map